backend/core/src/migration/1658871879940-multiselect-question.ts

```diff
@@ -322,6 +336,19 @@ export class multiselectQuestion1658871879940 implements MigrationInterface {
           juris
         )
         shouldPush = false
+      } else if (
+        !isForPreferences &&
+        (!formMetaData.type || formMetaData.type !== "checkbox") &&
+        index === formMetaData.options.length - 1
+      ) {
```

Comment:
These if statements might read a bit better if you condensed the same functionality together. You could either have an if else statement with `index !== formMetaData.options.length - 1` and then an inner if statement of the two other checks. 

Or you could do the inverse of `!isForPreferences &&
        (!formMetaData.type || formMetaData.type !== "checkbox")` being the top level if statement with the index check being the inner if. 
        
However, what you have here is ok, especially since it's a migration file.